js/main.js

```diff
@@ -142,3 +147,24 @@ $list.addEventListener('click', function (event) {
     document.querySelector('.title-text').textContent = 'Edit Entry';
   }
 });
+
+$deleteButton.addEventListener('click', function (event) {
+  $overlay.className = 'row align-center modal-background';
+});
+
+$cancelButton.addEventListener('click', function (event) {
+  $overlay.className = 'row align-center modal-background hidden';
+});
+
+$confirmButton.addEventListener('click', function (event) {
+  $targetedListItem.remove();
+  data.entries.splice(data.editing.entryId - 1, 1);
```

Comment:
This is a bad idea, you are using the ids for math. As soon as one entry gets deleted, this will stop working for any entries created after that id. You should be looping over the array to find the entry with the matching id.